echopype/echodata/convention/1.0.yml

```diff
@@ -89,6 +89,6 @@ variable_and_varattributes:
       standard_name: platform_heave_angle
       units: arc_degree
       valid_range: (-90.0, 90.0)
-    water_level:
+    range_offset:
       long_name: z-axis distance from the platform coordinate system origin to the sonar transducer
```

Comment:
In generalizing from `water_level` to `range_offset`, the long_name description "z-axis distance ..." doesn't seem to be strictly accurate any more. It seems to me that it should be something like "Along-range distance ..."